src/connection.js

```diff
@@ -694,9 +620,37 @@ class Connection extends EventEmitter {
   }
 
   getInitialSql() {
-    const xact_abort = this.config.options.abortTransactionOnError ? 'on' : 'off';
+    const enableQuotedIdentifier = this.config.options.enableQuotedIdentifier ? 'on' : 'off';
+    const enableArithAbort = this.config.options.enableArithAbort ? 'on' : 'off';
+    const enableNumericRoundabort = this.config.options.enableNumericRoundabort ? 'on' : 'off';
+    const enableAnsiWarnings = this.config.options.enableAnsiWarnings ? 'on' : 'off';
+    const enableAnsiPadding = this.config.options.enableAnsiPadding ? 'on' : 'off';
+    const enableAnsiNull = this.config.options.enableAnsiNull ? 'on' : 'off';
     const enableAnsiNullDefault = this.config.options.enableAnsiNullDefault ? 'on' : 'off';
-    return 'set textsize ' + this.config.options.textsize + '\nset quoted_identifier on\nset arithabort off\nset numeric_roundabort off\nset ansi_warnings on\nset ansi_padding on\nset ansi_nulls on\nset ansi_null_dflt_on ' + enableAnsiNullDefault + '\nset concat_null_yields_null on\nset cursor_close_on_commit off\nset implicit_transactions off\nset language us_english\nset dateformat mdy\nset datefirst 7\nset transaction isolation level ' + (this.getIsolationLevelText(this.config.options.connectionIsolationLevel)) + '\nset xact_abort ' + xact_abort;
+    const enableConcatNullYeldsNull = this.config.options.enableConcatNullYeldsNull ? 'on' : 'off';
+    const enableCursorCloseOnCommit = this.config.options.enableCursorCloseOnCommit ? 'on' : 'off';
+    const enableImplicitTransactions = this.config.options.enableImplicitTransactions ? 'on' : 'off';
+    const xact_abort = this.config.options.abortTransactionOnError ? 'on' : 'off';
+
+    var ret = 'set textsize ' + this.config.options.textsize + '\n' +
```

Comment:
Can you make this variable `const` here? Or remove the variable completely and just change it back to a `return`.

You could also change this string to make use of template literals to make it a bit more readable if you want.